benchmarking/run_synthetic_dataset.sh

```diff
@@ -3,7 +3,7 @@
 
 python src/evaluator.py \
 --mode synthetic \
---model-name "llama3-405b" \
+--model-names "llama3-8b llama3-70b	llama3-405b" \
```

Comment:
Let's put the sambastudio model names by default

Reply:
thanks! done!